src/components/AnonymousChat/child/ChannelImage.tsx

```diff
@@ -56,13 +57,13 @@ const ChannelImage = ({
       display: 'flex',
       justifyContent: 'center',
       alignItems: 'center',
-      backgroundColor: COLORS.anon_primary
+      backgroundColor: COLORS.anon_secondary
     },
     myPostNotificationImageContainer: {
-      backgroundColor: isAnonymousTab ? COLORS.anon_primary : COLORS.signed_secondary
+      backgroundColor: isAnonymousTab ? COLORS.anon_secondary : COLORS.signed_secondary
     },
     anonPmNotificationImageContainer: {
-      backgroundColor: COLORS.anon_primary
+      backgroundColor: COLORS.anon_secondary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here involve modifying the `backgroundColor` property for several style objects. As with the previous comment, ensure that `COLORS.anon_secondary` and `COLORS.signed_secondary` are defined and valid.
```diff
-      backgroundColor: COLORS.anon_primary
+      backgroundColor: COLORS.anon_secondary

-      backgroundColor: isAnonymousTab ? COLORS.anon_primary : COLORS.signed_secondary
+      backgroundColor: isAnonymousTab ? COLORS.anon_secondary : COLORS.signed_secondary

-      backgroundColor: COLORS.anon_primary
+      backgroundColor: COLORS.anon_secondary
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->